{"name": "test_get_tag_user[caseinfo5]", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestTag' object has no attribute 'logging'", "trace": "self = <test_case.gzh_manage.Test_tag.TestTag object at 0x0000000004A17E80>\ncaseinfo = {'count': 0, 'id': 106, 'name': '\\\\u7f8e\\\\u56fd\\\\u4eba1358'}\n\n    @pytest.mark.parametrize('caseinfo',read_yaml('test_case/gzh_manage/tag.yaml'))\n    def test_get_tag_user(self,caseinfo):\n        base=read_yaml('test_case/gzh_manage/get_tag_user.yaml')\n        url=base['request']['url']\n        method=base['request']['method']\n        data={'tagid':caseinfo['id']}\n        res=RequestsUtil.send_requests(method,url,data)\n>       self.logging.info(\"请求地址:{}\".format(url))\nE       AttributeError: 'TestTag' object has no attribute 'logging'\n\ntest_case\\gzh_manage\\Test_tag.py:45: AttributeError"}, "parameters": [{"name": "caseinfo", "value": "{'count': 0, 'id': 106, 'name': '\\\\u7f8e\\\\u56fd\\\\u4eba1358'}"}], "start": 1653271601883, "stop": 1653271601985, "uuid": "bd9f8424-921c-4676-b9ff-85fa38662e10", "historyId": "e5562346c0b5357a30de87971b80c939", "testCaseId": "3cd71c666f1092bdacbd19bcc40bbe2b", "fullName": "test_case.gzh_manage.Test_tag.TestTag#test_get_tag_user", "labels": [{"name": "parentSuite", "value": "test_case.gzh_manage"}, {"name": "suite", "value": "Test_tag"}, {"name": "subSuite", "value": "TestTag"}, {"name": "host", "value": "PC-20180915DYDK"}, {"name": "thread", "value": "88956-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.gzh_manage.Test_tag"}]}